examples/secret.py

```diff
@@ -0,0 +1,98 @@
+import typing
+
+import discord
+from discord.ext import commands
+
+bot = commands.Bot(command_prefix=commands.when_mentioned, description="Nothing to see here!")
+
+@bot.event
+async def on_command(ctx):
```

Comment:
```suggestion
async def on_command(ctx: commands.Context):
```
If this is going to be kept, be consistent with typing ctx.